src/views/Register.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div id="register">
+    <div class="container">
+      <form>
+            <h1>Signup</h1>
+            <hr>
+            <div>
+              <label for="username">First Name</label>
+              <input type="text" id="username"  v-model="userData.firstName">
+            </div>
+            <div>
+              <label for="password">Password</label>
+              <input type="password" id="password"  v-model="userData.password">
+              <p>
+                {{ userData.password }}
+              </p>
+            </div>
+        <hr>
+        <div class="row">
+            <button  @click.prevent="register">Submit!
```

Comment:
nog een rare indent? ook 2 spaces na <button hier

Reply:
ook raar dat de clossing tag new line is

Reply:
Done